tests/phpunit/api/v3/CampaignTest.php

```diff
@@ -26,6 +26,7 @@ public function setUp(): void {
       'created_date' => 'first sat of July 2008',
     ];
     parent::setUp();
+    \CRM_Core_BAO_ConfigSetting::enableComponent('CiviCampaign');
```

Comment:
Yea, this failed because `CampaignTest` neglected to enable the `CiviCampaign` component!